src/app/app-routing.module.ts

```diff
@@ -13,15 +14,22 @@ const routes: Routes = [
   },
   {
     path: ':resource/:uid',
-    component: StoryComponent
+    component: StoryComponent,
+    canActivate: [AuthGuardService]
   },
   {
     path: 'siblings/:resource/:uid',
-    component: SiblingsComponent
+    component: SiblingsComponent,
+    canActivate: [AuthGuardService]
   },
   {
     path: 'relationship/:resource/:uid/:relationship',
-    component: ArtifactRelationshipComponent
+    component: ArtifactRelationshipComponent,
+    canActivate: [AuthGuardService]
+  },
+  {
+    path: '**',
```

Comment:
It might be handy to add a comment describing why this redirect rule is necessary.